jdaviz/app.py

```diff
@@ -1005,7 +1005,7 @@ def return_unique_name(self, data_label, ext=None):
         return data_label
 
     def add_data_to_viewer(self, viewer_reference, data_label,
-                           clear_other_data=False):
+                           visible=True, clear_other_data=False):
```

Comment:
should we instead have `visible` at the end of kwargs?  This is in `app`, so shouldn't be considered as API-breaking, but if anywhere else relies on the positional order of `clear_other_data`, this could cause problems.

Reply:
As you stated, it is safer to put `visible` at the end. Is there a reason why it cannot be after `clear_other_data`?

Reply:
I just think that it intuitively makes sense to be in this order (`visible` seems more directly tied to the other inputs and action of the method, whereas `clear_other_data` is more loosely connected and probably less-often to be used).  But if API breaking concerns warrants switching the order, I can do that.